### ISSUE TITLE
📈 Config for `tmux`

### ISSUE THREAD

**Issue by Toliak** [Toliak/MakeLinuxEasier#3]
https://github.com/samoshkin/tmux-config

To install:
```
export TMUX_CONFIG_PATH="/tmp/change-it"

git clone https://github.com/exitstop/tmux-config --depth 1 $TMUX_CONFIG_PATH
/bin/bash $TMUX_CONFIG_PATH/install.sh
```

**Comment by Toliak** [comment]
Used this config
https://github.com/gpakosz/.tmux

**Comment by Toliak** [comment]
Closed in ecbb56c5435595b2e6a8849c4b936c000289142d
